src/features/broker-protection/actions/click.js

```diff
@@ -12,18 +12,25 @@ export function click (action, userData, root = document) {
     // there can be multiple elements provided by the action
     for (const element of action.elements) {
         const rootElement = selectRootElement(element, userData, root)
-        const elem = getElement(rootElement, element.selector)
+        const elements = getElements(rootElement, element.selector)
 
-        if (!elem) {
+        if (!elements?.length) {
             return new ErrorResponse({ actionID: action.id, message: `could not find element to click with selector '${element.selector}'!` })
         }
-        if ('disabled' in elem) {
-            if (elem.disabled) {
-                return new ErrorResponse({ actionID: action.id, message: `could not click disabled element ${element.selector}'!` })
+
+        const loopLength = element.multiple && element.multiple === true ? elements.length : 1
```

Comment:
not worth changing right now, but the double check here is redundant :)
```suggestion
        const loopLength = element.multiple === true ? elements.length : 1
```